### PR TITLE
Make it compatible w/ leaflet 1.2.0

### PR DESCRIPTION
**Pull request by hefler**
As code proposed on issue #58.

### CODE REVIEW

**Comment by hefler** [comment]
Re-identet, the issue before happened because I used git's own editor.

File: leaflet.textpath.js [hunk 1]
```diff
@@ -75,26 +77,26 @@ var PolylineTextPath = {
...
         text = text.replace(/ /g, '\u00A0');  // Non breakable spaces
         var id = 'pathdef-' + L.Util.stamp(this);
-        var svg = this._map._pathRoot;
+        var svg = this._map._renderer;
         this._path.setAttribute('id', id);
 
         if (options.repeat) {
             /* Compute single pattern length */
-            var pattern = L.Path.prototype._createElement('text');
+            var pattern = document.createElementNS('http://www.w3.org/2000/svg', 'text');
```
Comment (FoboCasteR) [inline leaflet.textpath.js]: L.SVG.create

File: leaflet.textpath.js [hunk 2]
```diff
@@ -4,6 +4,8 @@
  * http://mapbox.com/osmdev/2012/11/20/getting-serious-about-svg/
  */
 
+import L from 'leaflet';
```
Comment (FoboCasteR) [inline leaflet.textpath.js]: :scissors:

**Comment by mabhub** [comment]
Could you please try rebasing your branch to limit commits to meaningfull ones ?

*Not blocking but I'm just inquiring: are your code enhancements backward compatible with previous Leaflet versions (`>0.7.7 & <1.2.0)?*

And aslo, great :thumbsup: that you removed the `import L from 'leaflet'; ` which would have broken every usage without code transpiler like babel or webpack.

**Comment by hefler** [comment]
@mabhub I'm not super familiar with all git toolset (that includes rebase). I'm more into the bread and butter. 😅 
I believe that this change is not backwards compatible, what I've done was just gather the codes from issue #58. All creds to @visionbegin. I just fixed one or other stuff that was wrong as I wanted to use the plugin.

**Comment by mabhub** [comment]
I was trying to rebase your branch to latest development, and it turns out that the changes @hefler  proposed were already done on leaflet0.8-dev branch.

So I applied these changes on the main branch *(now `master` instead of `gh-pages`)* and proposed this in the PR #66 

@hefler : Please try this version (from PR or from master after PR validation) and tell us from your point of view if it resolves #58 

For now, I close current PR. Don't hesitate to comment if there is anything wrong.

**Pull request closed without merging**